src/components/WebexMeetingInfo/WebexMeetingInfo.jsx

```diff
@@ -39,7 +39,7 @@ export default function WebexMeetingInfo({className, meetingID, style}) {
     title,
   } = useMeeting(meetingID);
   const cssClasses = webexComponentClasses('meeting-info', className);
-  let infoComponent = <Spinner />;
+  let infoComponent = <Spinner style={{height: '36px', width: '36px'}} />;
```

Comment:
please add a `size={36}` prop to the component as well, which would modify the style prop:
```js
if (size) {
  style.width = size;
  style.height = size;
}
```
if eslint does not allow you to change the `style` var, make a copy:
```js
const cssStyle = {...style};
if (size) {
  cssStyle.width = size;
  cssStyle.height = size;
}
```
which is even better.